recipes/libmaxminddb/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+import os.path
+import shutil
+from conans import ConanFile, CMake, tools
+
+
+class libmaxminddbConan(ConanFile):
+    name = "libmaxminddb"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://maxmind.github.io/libmaxminddb/"
+    description = "C library for the MaxMind DB file format"
+    topics = ("MaxMind", "GeoIP")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "src"
```

Comment:
```suggestion
        return "source_subfolder"
```

This is a very common default.

Reply:
All right. Understood.